storybook/src/stories/UseArray/Burger/constants.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  BottomBun,
+  Cheese,
+  Ketchup,
+  Lettuce,
+  Mayo,
+  Mustard,
+  Patty,
+  Tomato,
+  TopBun,
+} from './components'
+import { Ingredient, IngredientGroup } from './types'
+
+const INGREDIENT_WIDTH = '20rem'
+
+export const INGREDIENTS = {
+  top: {
+    label: 'Top Bun',
+    svg: <TopBun width={INGREDIENT_WIDTH} />,
+    color: '#F4A335',
+  },
```

Comment:
This whole file can exclude jsx if you use the svg component here instead of the rendered element. Also let's you avoid repeating `width={INGREDIENT_WIDTH}`.

```suggestion
  top: {
    label: 'Top Bun',
    svg: TopBun,
    color: '#F4A335',
  },
```